specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ServerConfigurationOptions.json

```diff
@@ -0,0 +1,264 @@
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved managed instance server configuration options.",
+            "schema": {
+              "$ref": "#/definitions/ServerConfigurationOptionListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionNotFound - The requested subscription was not found.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets a list of managed instance server configuration options.": {
+            "$ref": "./examples/ServerConfigurationOptionList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/serverConfigurationOptions/{serverConfigurationOptionName}": {
+      "get": {
+        "tags": [
+          "ServerConfigurationOptions"
+        ],
+        "description": "Gets managed instance server configuration option.",
+        "operationId": "ServerConfigurationOptions_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "serverConfigurationOptionName",
+            "in": "path",
+            "description": "The name of the server configuration option.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "allowPolybaseExport"
+            ],
+            "x-ms-enum": {
+              "name": "ServerConfigurationOptionName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified managed instance server configuration option.",
+            "schema": {
+              "$ref": "#/definitions/ServerConfigurationOption"
+            }
+          },
+          "default": {
```

Comment:
>default [](http://example.com/codeflow?start=11&length=7)

provide default error response schema here

Reply:
@jamestao @ericshape I didn't manage to find other APIs under Microsoft.Sql that have default error response schema. Should we add it for this API?  If we should, can you tell me what should be done and how to generate it as I didn't manage to find any example

Reply:
@rkmanda same as other newly added swagger rules, we are planning to add it to our SQL RP's auto-generation process. But it needs effort and we will try our best to add it asap. Can we ask for the exception for 2022-08-01-preview? thanks~

Reply:
@OgnjenArsenijevic @ericshape @rkmanda it shouldn't require much effort to $ref the common error schema definition from common types in the swagger - can you fix your SQL RP autogeneration to just do that?

Reply:
@ericshape can comment if, and when it could be fixed, but as far as I'm aware, I think that the plan is to release API version 2022-08-01 in the week after Thanksgiving. So, in order to make sure that our API will be included in 2022-08-01 API version this PR should be merged as soon as possible (in next couple of days), so I don't think that fix can be made in time to be included in this PR.

After fix is added in the future I suppose that it would be possible to make new PR and then add error schema definition for this and all other APIs that already exist.

Reply:
oh, @TimLovellSmith  I replied in another thread:
https://github.com/Azure/azure-rest-api-specs/pull/21415#discussion_r1026792600